lms/djangoapps/course_api/blocks/transformers/load_override_data.py

```diff
@@ -0,0 +1,91 @@
+"""
+Load Override Data Transformer
+"""
+from datetime import datetime
+import json
+from dateutil.parser import parse
+
+from openedx.core.djangoapps.content.block_structure.transformer import BlockStructureTransformer
+
+from courseware.models import StudentFieldOverride
+
+
+class OverrideDataTransformer(BlockStructureTransformer):
+    """
+    A transformer that load override data in xblock.
+    """
+    WRITE_VERSION = 1
+    READ_VERSION = 1
+    requested_fields = [
+        'start', 'display_name', 'type', 'due', 'graded', 'special_exam_info', 'format', 'grading_policy'
+    ]
```

Comment:
I think we should shorten this list. I understand this is a generic block transformer (like the override is generic), but I don't want to suggest to the next developer that it's safe to override `graded` or `grading_policy` for an individual learner. 

Let's limit this to `'start', 'display_name', 'due',` for now and then leave a comment that the list of fields are in support of Indvidual due dates and could be expanded for other use cases.